doc/source/configuration.rst

```diff
@@ -0,0 +1,292 @@
+
+.. code-block:: yaml
+
+  clouds:
+    otc:
+      auth:
+        password: '<PASSWORD>'
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
+      sk: '<SK_VALUE>'
+
+
+.. _environment-variables:
+
+Agency based authorization
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open Telekom Cloud supports a concept of agencies. One domain delegates access
+to resources to another domain. After trust relationship is established the
+following configuration can be used in ``clouds.yaml``:
+
+.. code-block:: yaml
+
+  clouds:
+    otc:
+      profile: otc
+      auth_type: agency
+      auth:
+        username: '<USER_NAME>'
+        project_name: '<eu-de_project>'
+        # or project_id: '<123456_PROJECT_ID>'
+        user_domain_name: 'OTC00000000001000000xxx'
+        # or user_domain_id: '<123456_DOMAIN_ID>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+        target_domain_id: '<123456_DOMAIN_ID>' # Domain where agency is created
+        # or target_domain_name: '<123456_DOMAIN_NAME'
+        target_agency_name: 'test_agency' # name of the agency
+        target_project_name: '<123456_PROJECT_NAME>' # project scoped operations
+        # or target_project_id: '<123456_PROJECT_ID>'
+        # When target_project_xx is not set - domain scope is selected
+
+Configuration of Environment Variables
+--------------------------------------
+
+Historically OpenStack tools are supporting configuration through environment
+variables. Create a simple file like ``.ostackrc`` in the home directory and
+source the file to make the variables available. On Open Telekom Cloud servers
+this file exists on bootup and needs to be changed according to your credentials.
+
+.. code-block:: bash
+
+    # .ostackrc file
+    export OS_USERNAME="<USER_NAME>"
+    export OS_USER_DOMAIN_NAME=<OTC00000000001000000XYZ>
+    export OS_PASSWORD=<PASSWORD> # optional
+    export OS_TENANT_NAME=eu-de
+    export OS_PROJECT_NAME=<eu-de_PROJECT_NAME>
+    export OS_AUTH_URL=https://iam.eu-de.otc.t-systems.com:443/v3
+    export NOVA_ENDPOINT_TYPE=publicURL
+    export OS_ENDPOINT_TYPE=publicURL
+    export CINDER_ENDPOINT_TYPE=publicURL
+    export OS_VOLUME_API_VERSION=2
+    export OS_IDENTITY_API_VERSION=3
+    export OS_IMAGE_API_VERSION=2
+
+Run the source command to make the ``environment variables`` available.
+
+.. code-block:: bash
+
+   $ source .ostackrc
+
+The ``environment variables`` are now available for usage. For testing your
+connection run the following command.
+
+Test your connection
+^^^^^^^^^^^^^^^^^^^^
+
+If you followed the `installation advices <index>`_ for your specific
+operating system, you can use the following command to test the basic
+functionality.
+
+.. code-block:: bash
+
+    $ openstack flavor list
+
+.. note::
+   You don't need to specify the `--os-cloud` parameter when environment
+   variables are used.
+
+Cache Settings
+--------------
+
+Accessing a cloud is often expensive, so it's quite common to want to do some
```

Comment:
I suggest the following phrasing: '[...]quite common to use client-side caching[...]'

Reply:
thanks, it's a nice advice.